etc/rose-meta/rose-suite-info/rose-meta.conf

```diff
@@ -23,21 +23,12 @@ help=Specify a space-separated list of users apart from the owner who have
 sort-key=02-users-0
 
 [=description]
+compulsory=true
```

Comment:
The problem here is that all existing suites will fail to validate. I think we need to drop the requirement that this is compulsory. It can be added by default, but it shouldn't be compulsory.